octobot_trading/modes/modes_factory.py

```diff
@@ -88,6 +87,16 @@ async def create_trading_mode(trading_mode_class: modes.AbstractTradingMode.__cl
         raise e
 
 
+def create_temporary_trading_mode_with_local_config(
```

Comment:
What is the use case?

Reply:
this https://github.com/Drakkar-Software/OctoBot-Trading/pull/738#discussion_r985087531

Reply:
Yes